templates/api-template-java/pom.xml

```diff
@@ -21,7 +21,7 @@
 	<dependency>
 		<groupId>com.fasterxml.jackson.core</groupId>
 		<artifactId>jackson-databind</artifactId>
-		<version>2.9.9.1</version>
+		<version>2.10.0.pr2</version>
```

Comment:
Is this a valid stable version?

Reply:
Not aware of this

Reply:
This came in as part of Snyk autofix. Safe to merge.